pkg/util/log/format_crdb_v2.go

```diff
@@ -447,3 +455,257 @@ func (buf *buffer) maybeMultiLine(
 var startRedactionMarker = string(redact.StartMarker())
 var endRedactionMarker = string(redact.EndMarker())
 var markersStartWithMultiByteRune = startRedactionMarker[0] >= utf8.RuneSelf && endRedactionMarker[0] >= utf8.RuneSelf
+
+var (
+	entryREV2 = regexp.MustCompile(
+		`(?m)^` +
+			/* Severity                 */ `(?P<severity>[` + severityChar + `])` +
+			/* Date and time            */ `(?P<datetime>\d{6} \d{2}:\d{2}:\d{2}.\d{6}) ` +
+			/* Goroutine ID             */ `(?:(?P<goroutine>\d+) )` +
+			/* Go standard library flag */ `(?P<gostd>\(gostd\) )?` +
+			/* Channel                  */ `(?:(?P<channel>\d+)@)?` +
+			/* File                     */ `(?P<file>[^:]+):` +
+			/* Line                     */ `(?:(?P<line>\d+) )` +
+			/* Redactable flag          */ `(?P<redactable>(?:` + redactableIndicator + `)?) ` +
+			/* Context tags             */ `(?:\[(?P<tags>(?:[^]]|\][^ ])+)\] )` +
+			/* Counter                  */ `(?P<counter>(?:\d+)?) ` +
+			/* Continuation marker      */ `(?P<continuation>[ =!+|])` +
+			/* Message                  */ `(?P<msg>.*)$`,
+	)
+	v2SeverityIdx     = entryREV2.SubexpIndex("severity")
+	v2DateTimeIdx     = entryREV2.SubexpIndex("datetime")
+	v2GoroutineIdx    = entryREV2.SubexpIndex("goroutine")
+	v2GoStdIdx        = entryREV2.SubexpIndex("gostd")
```

Comment:
Seems like we're not using this. Does it mean we lose this information? Should we store it somewhere?

Reply:
I created a group for the go standard library flag so that the parser would ignore it when present. I do not think we need to store it? Maybe @knz has some thoughts on this. For now I can remove that line.